docs/docs/providers/idme.md

```diff
@@ -0,0 +1,48 @@
+---
+id: idme
+title: ID.me
+---
+
+## Documentation
+
+https://developers.id.me/documentation
+
+## Configuration
+
+https://developers.id.me/organizations
+
+## Options
+
+The **ID.me Provider** comes with a set of default options:
+
+- [ID.me Provider options](https://github.com/nextauthjs/next-auth/blob/main/packages/next-auth/src/providers/idme.ts)
+
+You can override any of the options to suit your own use case.
+
+## Example
+
+```js
+import IDMeProvider from "next-auth/providers/idme";
+...
+providers: [
+  IDMeProvider({
+    clientId: process.env.IDME_ID,
+    clientSecret: process.env.IDME_SECRET,
+    authorization: {
+      params: {
+        scope: 'openid military teacher',
+        redirect_uri: `${process.env.NEXTAUTH_URL}/api/auth/callback/idme`
```

Comment:
```suggestion
```

This is not needed.

Reply:
Just checked and you are correct. Will remove from doc's.